MDANSE/Src/MDANSE/Framework/AtomSelector/atom_selection.py

```diff
@@ -0,0 +1,76 @@
+#    This file is part of MDANSE.
+#
+#    MDANSE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from typing import Set, Sequence
+
+from MDANSE.MolecularDynamics.Trajectory import Trajectory
+
+
+def select_atoms(
+    trajectory: Trajectory,
+    *,
+    index_list: Sequence[int] = None,
+    index_range: Sequence[int] = None,
+    index_slice: Sequence[int] = None,
```

Comment:
Structly pep-484 doesn't allow implicit `None`.

C.f. the end of https://peps.python.org/pep-0484/#union-types